docs/building-a-form/common-definitions.md

```diff
@@ -2,27 +2,29 @@
 
 # Common definitions
 
-Definitions are pieces of the form config that can be dropped in to represent specific types of questions. Most often used in `uiSchema`, definitions include features such as label text, validation functions, error messages, and rules for which widget to render.
+Definitions, located in [/src/js/definitions](../../src/js/definitions), are pieces of the form config that can be dropped in to represent specific types of questions. They include features such as label text, validation functions, error messages, and rules for which widget to render.
 
-There are common types of definitions: `schema`/`uiSchema` objects and functions that return `schema`/`uiSchema` objects. For the function versions, there is documentation within the fields for the parameters. Definitions are located in [/src/js/definitions](../../src/js/definitions).
-
-- Simple definitions are provided as `schema` and `uiSchema` objects that you can import and overwrite to customize.
-- More complex definitions are functions that require certain parameters.
+Definitions export `schema` and `uiSchema` objects, which are functions that require certain parameters.
 
 ### Using definitions
 
 To use a definition, import it near the top of the file you want to reference it in:
 
 ```js
-import currencyUI from 'us-forms-system/lib/js/definitions/currency';
+import currencyConfig from 'us-forms-system/lib/js/definitions/currency';
 ```
 
-Then, call it to add all the `uiSchema` definitions. In this example, the definition is a function that takes the title for that field, which is used to populate the 'ui:title' property in uiSchema:
+Then, call it to add the `schema` and `uiSchema` functions. The definition is a function that takes the title for that field, which is used to populate the 'ui:title' property in uiSchema:
```

Comment:
I think we need to change this language a little bit. It's still pretty specific to `uiSchema`. Here's my recommended edit:

Then, use the definition by calling the `schema` and `uiSchema` functions on it where needed. Sometimes the functions take arguments to configure how the definitions are used. In this particular case, the `uiSchema` for currency takes an optional argument for the title of the field. In this example we want to set the title to "Currency", so we pass that as a string to the `uiSchema` function.

Reply:
Thank you!